### PR TITLE
Pull SSL Certificate and Key directly from S3

### PR DESCRIPTION
**Pull request by JordanGuinn**
Necessary for us to pull said certificates and keys out of source control!

------------------------------------------------------------------------------------------------------------------------

### Associated PRs:
- https://github.com/skylight-hq/ctoec-devops/pull/132
- https://github.com/ctoec/winged-keys/pull/59
- https://github.com/ctoec/ecis-experimental/pull/1277

### CODE REVIEW

**Review by bamader — APPROVED** [review]
Awwwwww yeeeeeeeah

File: deploy.sh [hunk 1]
```diff
@@ -27,16 +27,25 @@ then
    exit 1
 fi
 
-echo "Installing yarn dependencies..."
-yarn install --frozen-lockfile
-cd client && yarn install --frozen-lockfile --network-concurrency 1
+echo "Clearing out all old dependencies..."
+rm -rf node_modules
```
Comment (kmagida110) [inline deploy.sh:31]: Good fixes here!

File: rebuild.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+echo "Building the React app..."
+yarn build
+
+echo "Installing server dependencies..."
+cd ../
+yarn install --frozen-lockfile
+
+echo "Building the Express server..."
+yarn build
```
Comment (kmagida110) [inline rebuild.sh:19]: Does this not need the stage name?

**Review by kmagida110 — APPROVED** [review]
A few non-blocking comments

File: rebuild.sh (hunk 2 above)
Comment (JordanGuinn) [reply rebuild.sh:19]: Oh I put this up specifically for local development (where stage isn't needed) - I'll outline as much in the comments of the script

**Pull request merged**